### PR TITLE
Updated ARCH links with a new link to NIST SP 800-57

### LINKED ISSUE #501: Outdated NIST link in V1
The V1 has an outdated link to NIST SP 800-57
Old link: http://csrc.nist.gov/publications/nistpubs/800-57/sp800-57-Part1-revised2_Mar08-2007.pdf
New link: https://csrc.nist.gov/publications/detail/sp/800-57-part-1/rev-5/final
Or a new direct pdf link: https://nvlpubs.nist.gov/nistpubs/SpecialPublications/NIST.SP.800-57pt1r5.pdf

### PR DESCRIPTION
**Pull request by julepka**
Updated the link to NIST SP 800-57. Now it points not to the exact version but the `/final` one. It should not get outdated as fast as previous links. 

- [x] Your PR is linked to an issue.
- [x] Your PR follows the [contribution guidelines](https://github.com/OWASP/owasp-masvs/blob/master/CONTRIBUTING.md "Contribution guidelines")
- [ ] All links are valid and use HTTPS and have been formatted as [TEXT](URL “NAME”) - _I think it will not be consistent to do that_

This PR closes #501 .

### CODE REVIEW

**Comment by cpholguera** [comment]
Thanks @julepka!

FYI I've changed the text above to be: "This PR closes #501 ."
"closes" + issue ID acts as a magic keyword. You can see the Issue now linked in the right side of the page and once we merge the issue will be automatically closed ;)

Feel free to keep using that :)

**Review by cpholguera — CHANGES_REQUESTED** [review]
Could you please fix it for all other languages? Thanks in advance!

https://github.com/OWASP/owasp-masvs/search?q=sp800-57-Part1-revised2_Mar08-2007&type=

File: Document-zhtw/0x06-V1-Architecture_design_and_threat_modelling_requireme.md [hunk 1]
```diff
@@ -33,4 +33,5 @@
 - OWASP Thread modelling - <https://owasp.org/www-community/Application_Threat_Modeling>
 - OWASP Secure SDLC Cheat Sheet - <https://github.com/OWASP/CheatSheetSeries/blob/master/cheatsheets_excluded/Secure_SDLC_Cheat_Sheet.md>
 - Microsoft SDL - <https://www.microsoft.com/en-us/sdl/>
-- NIST SP 800-57 - <http://csrc.nist.gov/publications/nistpubs/800-57/sp800-57-Part1-revised2_Mar08-2007.pdf>
+- NIST SP 800-57 - <https://csrc.nist.gov/publications/detail/sp/800-57-part-1/rev-5/final>
+- security.txt - <https://securitytxt.org/>
```
Comment (julepka) [inline Document-zhtw/0x06-V1-Architecture_design_and_threat_modelling_requireme.md:37]: I've noticed that zhtw translation is missing this link that is present in all other languages, so, I've added it as well

**Review by cpholguera — APPROVED** [review]
Thank you very much again @julepka!

**Pull request merged**